app.rb

```diff
@@ -250,6 +272,11 @@ def biometric_comparison_required_value(ial)
       ial == 'biometric-comparison-required'
     end
 
+    def enhanced_ipp_required_value(ial)
```

Comment:
this is returning a boolean so let's make it a `?` method, and maybe even rename it to clearer about what is being checked?
```suggestion
    def requires_enhanced_ipp?(ial)
```

Reply:
resolved with commit 6c529a1fc3f76e9897765cc2b0d92928a0c48e74